src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -175,6 +181,7 @@ export interface IChartContainerProps {
 @observer
 export class ChartContainer extends React.Component<IChartContainerProps, {}> {
     private chartHeaderHeight = 20;
+    private mutationPlotRef: any;
```

Comment:
we should avoid `any` and be more specific here.

Reply:
Updated with the component name.